examples/ncursesInt/RF24Gateway_ncursesInt.cpp

```diff
@@ -653,6 +653,8 @@ void drawMeshPad()
         mvwprintw(meshPad, i + 1, 0, " Address: 0%o ID: %d \n", mesh.addrList[i].address, mesh.addrList[i].nodeID);
     }
 
+    meshScroll = rf24_max(meshScroll, 0);
```

Comment:
In this example too
https://github.com/nRF24/RF24Gateway/blob/bbd2b1068e46d5852bab55f8bbeac325ee9a0f4d/examples/ncursesInt/RF24Gateway_ncursesInt.cpp#L327-L332